feat(blog): add article about things that happened in asyncapi in Q1 (#656)
asyncapi/website: pages/blog/2022Q1-summary.md

---

> title: "Summary of things that happened around the AsyncAPI Initiative until end of April 2022"

Reviewer: ```suggestion
title: "Summary of things that happened at AsyncAPI Initiative from January to April, 2022"
```

date: 2022-04-28T06:00:00+01:00
type: Communication
tags:
  - Project Status
cover: /img/posts/2022Q1-summary/cover.webp
authors:
  - name: Lukasz Gornicki
    photo: /img/avatars/lpgornicki.webp
    link: https://twitter.com/derberq

>     byline: AsyncAPI Gardner

Reviewer: Gardner?

Author: ups 😄 was supposed to be Gardener 🤣

> excerpt: 'New AsyncAPI brand. AsyncAPI 2.4 and work on 3.0 spec release. Meetings organization changes. Conference organization. AND SO MUCH MORE!'

Reviewer: ```suggestion
excerpt: 'New AsyncAPI brand. AsyncAPI 2.4 and work on the 3.0 spec release. Meetings organization change. 2022 conference organization. AND SO MUCH MORE!'
```

featured: true
---

> TODO here will be link to podcast for those that do not read articles

> Welcome to another update for AsyncAPI Initiative. In this article, I want to highlight the most important things around AsyncAPI in the first few months of 2022.

Reviewer: ```suggestion
Welcome to another update from the AsyncAPI Initiative. In this article, I want to highlight the most important items contributed to AsyncAPI during the first few months of 2022.
```

> I usually ask the community if folks have anything important that I should put in my updates. Still, it is a pretty much subjective article based mainly on things I see in my personal bubble :smiley:

Reviewer: ```suggestion
I usually ask the community if folks have anything important that I should put in my updates. Still, this is a pretty subjective article because it's based mainly on things I see in my personal bubble. :smiley:
```

## Brand Refresh

> This one is pretty easy to spot, especially if you have been with AsyncAPI Initiative for some time :smiley:

Reviewer: ```suggestion
This one is pretty easy to spot, especially if you have been with AsyncAPI Initiative for some time. :smiley:
```

> 10th of March, we announced a change in the AsyncAPI brand. Change that was needed. A refresh of this faded green that I personally never liked :joy:

Reviewer: ```suggestion
On March 10, we announced a change to the AsyncAPI brand. Change that was needed. A refresh of this faded green that I personally never liked. :joy:
```

<TwitterTweetEmbed
  tweetId='1501977719915098120'
  options={{
    cards: 'hidden',
    width: 200,
    maxWidth: 400
  }}
/>

> It was a gigantic effort by [Missy Turco](https://twitter.com/missyturco). Months of work, research, and discussions with the community.

Reviewer: ```suggestion
It was a massive effort led by [Missy Turco](https://twitter.com/missyturco). Months of work, research, and discussions with the community.
```

@derberg  Here I go again, trying to add the word `led` or `leadership`.  😈

It is not all. More work in the area is needed:

> - Design System :muscle: - Get involved in [related repository](https://github.com/asyncapi/design-system) if you are interested in helping out. 

Reviewer: ```suggestion
- Design System :muscle: - Get involved in the [related design system repository](https://github.com/asyncapi/design-system) if you are interested in helping out. 
```
This is another accessibility example, where we want to use more specific text in our anchor text.

- AsyncAPI Mascots refresh - We need to refresh our existing Eve and Chan mascots to match the new brand. Any help would be appreciated.

> For more regular news and updates on topics around brand and design in AsyncAPI Initiative, I definitely recommend you to follow [Missy Turco](https://twitter.com/missyturco) and read her [regular updates on the subject](https://missyturco.notion.site/03556522990e4067ae877fd236c37268?v=d60154483de849cd8125f8333cfe1d36).

Reviewer: ```suggestion
For more regular news and updates on topics around brand and design in AsyncAPI Initiative, I definitely recommend you follow [Missy Turco](https://twitter.com/missyturco) and read her [regular design updates](https://missyturco.notion.site/03556522990e4067ae877fd236c37268?v=d60154483de849cd8125f8333cfe1d36).
```
This is another accessibility example, where we want to use more specific text in our anchor text.

## AsyncAPI Conference

> We want to organize the first conference that you not only can watch online but also participate in-person. May is right behind the cornern, not much time left. We need to start working on the event actively now.

Reviewer: ```suggestion
We want to organize our first hybrid conference, one you can watch online and participate in person. May is right behind the corner, so not much time is left. We need to start working actively on the event now.
```

> We discuss all the aspects of the conference [in Community Discussions](https://github.com/asyncapi/community/discussions/categories/asyncapi-conf-2022-organization) openly. Please join and actively support the effort. We need lots of volunteers to make that happen. 

Reviewer: ```suggestion
We discuss all aspects of our upcoming conference in our [Community Discussion for 2022 Conference Organization](https://github.com/asyncapi/community/discussions/categories/asyncapi-conf-2022-organization) openly. Please join and actively support the effort. We need lots of volunteers to make this happen. 
```
This is another accessibility example where we want to use more specific text in our anchor text.

> We need to have a special task force that will focus on the subject. Please join the discussion on [hosting a conference-dedicated weekly meeting](https://github.com/asyncapi/community/discussions/322).

Reviewer: ```suggestion
We need to have a special task force that will focus on the subject. Please join the [discussion on hosting a conference-dedicated weekly meeting](https://github.com/asyncapi/community/discussions/322).
```

Wanted our anchor text to be even more specific by clarifying (to screenreader users) that it's a GH Discussion link.

## Meetings scheduling automation and related changes

> Because of the community's growth and the growing number of topic-specific meetings, we had to spend some time improving things, and automating as much as possible. After some initial discussions like [this one about Google-services](https://github.com/asyncapi/community/discussions/236), some initial requirements were defined, called [Meeting as a Service](https://github.com/asyncapi/community/issues/245).

Reviewer: ```suggestion
Because of the community's growth and the growing number of topic-specific meetings, we had to spend some time improving things and automating as much as possible. After some initial discussions, such as [this discussion about Google-services](https://github.com/asyncapi/community/discussions/236), some initial requirements were defined, called [Meeting as a Service](https://github.com/asyncapi/community/issues/245).
```

Improving anchor text for accessibility.

> - A week ago, we stopped using our `asyncapi-users` Google Group. We removed it, and all the members were migrated as subscribers of the AsyncAPI Newsletter. This is the place where we will now send email notifications every Monday-morning with a list of upcoming meetings,

Reviewer: ```suggestion
- A week ago, we stopped using our `asyncapi-users` Google Group. We removed it, and all the members were migrated as subscribers to the AsyncAPI Newsletter. This newsletter is the new place where we'll send email notifications every Monday morning with a list of upcoming meetings.
```

> - We switched to the new Google Calendar instance managed by the official AsyncAPI Google account. You should remove/unsubscribe from the old calendar and [subscribe to the new one](https://calendar.google.com/calendar/u/3?cid=Y19xOXRzZWlnbG9tZHNqNm5qdWh2YnB0czExY0Bncm91cC5jYWxlbmRhci5nb29nbGUuY29t),

Reviewer: ```suggestion
- We switched to the new Google Calendar instance managed by the official AsyncAPI Google account. You should remove/unsubscribe from the old calendar and [subscribe to the new calendar](https://calendar.google.com/calendar/u/3?cid=Y19xOXRzZWlnbG9tZHNqNm5qdWh2YnB0czExY0Bncm91cC5jYWxlbmRhci5nb29nbGUuY29t),
```
 Improving anchor text for accessibility.

- [Our Technical Steering Committee (TSC)](https://github.com/asyncapi/community/discussions/295) approved to sponsor 8 Zoom licenses to host official AsyncAPI meetings. So far, only 3 licenses have been assigned. We have lots of room for more meetings and hosts, if there is a need.

> Huge thanks to [Krishna Kumar](https://github.com/Krishks369) and [Shreyansh Jain](https://github.com/sudoshreyansh) that helped in research related to the implementation :heart:

Reviewer: ```suggestion
Huge thanks to [Krishna Kumar](https://github.com/Krishks369) and [Shreyansh Jain](https://github.com/sudoshreyansh), who helped research the implementation. :heart:
```

> If you want to learn more about becoming a host of the meeting, or other things related to meetings, get familiar with our [dedicated meetings FAQ](https://github.com/asyncapi/community/blob/master/MEETINGS_ORGANIZATION.md).

Reviewer: ```suggestion
If you want to learn more about becoming a meeting host, or something else related to meetings, get familiar with our [dedicated meetings FAQ](https://github.com/asyncapi/community/blob/master/MEETINGS_ORGANIZATION.md).
```

> Oh, and I almost forgot, look at this :point_down: :star: 

Reviewer: ```suggestion
Oh, and I almost forgot, look at this. :point_down: :star: 
```

> All [info about AsyncAPI-related meetings](https://www.asyncapi.com/community/meetings) in one beatifull place :heart:

Reviewer: ```suggestion
All [info about AsyncAPI-related meetings](https://www.asyncapi.com/community/meetings) is in one beautiful place! :heart:
```

<Figure
  src="/img/posts/2022Q1-summary/meetings.webp"
  className="text-center"
  caption="View of dedicated community/meetings page on the AsyncAPI website."
/>

## Upcoming 2.4 and 3.0 releases of the spec

> We released a new [2.4.0 version of AsyncAPI Specification](https://github.com/asyncapi/spec/releases/tag/v2.4.0). There are many good features, like the possibility to provide the `id` of the Message unique for the entire AsyncAPI file or the ability to specify security valid only for a given operation. You definitely need to read [release notes](https://www.asyncapi.com/blog/release-notes-2.4.0).

Reviewer: ```suggestion
We released a [new 2.4.0 version of AsyncAPI Specification](https://github.com/asyncapi/spec/releases/tag/v2.4.0). There are many good features, like the possibility to provide a unique Message `id` for the entire AsyncAPI file or the ability to specify that security is valid only for a given operation. You definitely need to read the[release notes](https://www.asyncapi.com/blog/release-notes-2.4.0).
```

> Huge shout out to [Sergio Moya](https://twitter.com/smoyac), who was a release coordinator for 2.4 release and pushed all the doors to make sure the release could go out.

Reviewer: ```suggestion
Huge shout out to [Sergio Moya](https://twitter.com/smoyac), who was a release coordinator for the 2.4.0 release and pushed all the doors to make sure the release could go out.
```

> 2.4.0 was released, but this doesn't block us from working on something much bigger, 3.0.0 version of the specification :rocket:

Reviewer: ```suggestion
The release of 2.4.0 helps us continue working toward something much bigger; the 3.0.0 version of the specification! :rocket:
```

Author: 2.4 and 3.0 are not related, so 2.4 doesn't "help" working on 3.0

I just wanted to highlight that regular 2.x releases do not mean that we do not think on more significant changes that will come in 3.0

Reviewer: The reason why I use the word help is because now that you are done working on one project you have more time to do the next one ;)

I assure you it was not because one project relied on the other ✨😻🥳

Author: > help is because now that you are done working on one project you have more time to do the next one

but that is the thing, both are super not related, for example, Jonas works on 3.0 and has basically 0 to do with 2.4

Reviewer: I hear you; you're telling me that you have a concern that the wording will sound like they are related.

Totally get it, you can disregard this one if you like!

Reviewer: oh but still **add a punctuation mark at the end** tho, that was missing :)

Author: done, `!` added, thanks 🙏🏼

> If you do not know yet about it, I highly recommend you to have a look at [this article](https://www.asyncapi.com/blog/async-api-spec-3.0-release) from [Harsh Mishra](https://www.linkedin.com/in/harshcasper/).

Reviewer: ```suggestion
If you haven't heard about 3.0.0 yet, I highly recommend you have a look at the [3.0.0 release article](https://www.asyncapi.com/blog/async-api-spec-3.0-release) written by [Harsh Mishra](https://www.linkedin.com/in/harshcasper/).
```

> Please join these efforts, as there is much work ahead of us, but not many hands doing actual work. We meet every two weeks to sync the work efforts and discuss important topics. Feel free to join :pray:

Reviewer: ```suggestion
Please join these efforts, as there is still much work ahead but not enough hands to do all the work. We meet every two weeks to sync the work efforts and discuss important topics. Feel free to join. :pray:
```

The original way it was phrased sounded too harsh/negative. It came across as guilt-tripping the audience CAUSE CLEARLY NOT ENOUGH PEOPLE CARE. 🤣 So I re-phrased it. 😇

Author: well that was my intention and I want to leave it this way.
we only hear complaints, but no active work

Reviewer: Yeah I know that was your intention but I don't think that releasing a blog post with negative emotion is the most professional. I think you can express frustration without sounding negative or like we are guilt tripping the community.

If this was a personal blog or social media account then I wouldn't ask about this, I'm just pausing because of the fact that this is from the official AsyncAPI blog. 🤔

Author: this is from an official blog post but with author name on it. As long as it is following Code of Conduct, it is fine. And I don't think it is against CoC. Also I write in first paragraph that it is all from my personal perspective

Reviewer: okidoki :) 

go forth and be negative 🤣 ❤️

Author: ![](https://media.giphy.com/media/i8UwDjybLTjMO2Eh6h/giphy.gif)

Reviewer: ME TOO, especially when the stars align ⭐ ⭐ ⭐

> I also recommend following [Sergio Moya](https://twitter.com/smoyac) and reaching for [his regular updates on specification-related efforts](https://gist.github.com/smoya)

Reviewer: ```suggestion
I also recommend following [Sergio Moya](https://twitter.com/smoyac) and reaching for his [regular updates on specification-related efforts](https://gist.github.com/smoya)
```
I moved `his` outside of the anchor text because if a screen reader user is "scanning" only the links (they have a feature that allows them to click link by link), I don't think that it will be clear what's the **antecedent** of the pronoun **his**. So I think a screen reader user will feel it's clearer to have the link only say _"regular updates on specification-related efforts"_.

CAVEAT: The BEST anchor text for that situation would be to include `Sergio` vs `his`, but you will probably feel it sounds too repetitive to read the name Sergio a second time in this sentence. This is why I thought it was better to just keep the pronoun "his" outside of the anchor text. The anchor text is still very specific and clear, even without including the author's name.

## Contributors onboarding initiatives

> One of [goals for community-building efforts for 2022](https://github.com/asyncapi/community/discussions/193) was to involve AsyncAPI Initiative as a mentoring organization in as many coding (but not only) events. We have many maintainers here who like to help first-time contributors land their first pull requests in an open and relaxed environment.

Reviewer: ```suggestion
One of the [goals for our 2022 community-building efforts](https://github.com/asyncapi/community/discussions/193) was to involve AsyncAPI Initiative as a mentoring organization in as many coding (but not only) events as possible. We have many maintainers here who like to help first-time contributors land their first pull requests in an open and relaxed environment.
```

### Google Summer of Code

> We did not start well. We were not accepted for [Google Summer of Code](https://summerofcode.withgoogle.com). Don't ask me why. I really do not know. I wish I could have a clear answer from them about what failed on our end :sad:

Reviewer: ```suggestion
We did not fare well; we were not accepted for [Google Summer of Code 2022](https://summerofcode.withgoogle.com). Don't ask me why. I really do not know. I wish I could have a clear answer from them about what failed on our end. :sad:
```
I just have 1 thought.. the word **fail** is very strong and we may want to think about a more accurate way to describe what (probably?) happened.  What I mean is that the AsyncAPI GSoC application didn't necessarily fail, it could simply have been in tough competition due to the **amount** of OSS orgs that applied. 

We already know that a humongous company like Google easily gets BASICALLY THE WHOLE WORLD to apply to their programs, so if they got 2000+ applications and decided to only accept whatever number of applicants in 2022... odds are we were only one of a gazillion applications.

I would consider re-phrasing this the first part of this section so that it doesn't influence the community to feel too negative about getting a no this year.

Author: - `fare well` - doesn't it mean something like good bye? what I meant by `we did not start well` was that we did not start well, like a false start in a race :) 
- I modified a bit the `fail` part but left the word. I just made it more personal, that I think it is my failure. If in one place we say that someone succeeded with something, it is ok to also personally admit that I think I failed with GSoC application. If I blame others, I will not improve. Now I know that next year we need to work on official application in a group, to make sure we write it better. Hope you know what I mean

> We never give up here at AsyncAPI, though :smiley:

Reviewer: ```suggestion
We never give up here at AsyncAPI, though. :smiley:
```

### OpenForce

> March was a month where we joined [OpenForce](https://www.asyncapi.com/blog/openforce-2022) thanks to amazing support from [Hargun Kaur](https://www.linkedin.com/in/hkaur008/). As a result, we managed to onboard a few new folks and gained important contributions in exchange. There are actually a few PRs still open, but for now, completed are:

Reviewer: ```suggestion
March was a month where we joined [OpenForce](https://www.asyncapi.com/blog/openforce-2022) thanks to amazing support from [Hargun Kaur](https://www.linkedin.com/in/hkaur008/). As a result, we managed to onboard a few new folks and gained important contributions in exchange. There are  a few PRs still open, but here are some of the completed PRs:
```

> - [Namya LG](https://github.com/Namyalg) introduced a great improvement to our CI by creating dedicated workflows to [search for broken links](https://github.com/asyncapi/.github/issues/92) in our documentation across all repositories.

Reviewer: ```suggestion
- [Namya LG](https://github.com/Namyalg) introduced a great improvement to our CI by creating dedicated [CI workflows that search for broken links](https://github.com/asyncapi/.github/issues/92) in our documentation across all repositories.
```
Improving anchor text for accessibility.

> - [Harsh Mishra](https://www.linkedin.com/in/harshcasper/) introduced a workflow in our CI to [validate docker images](https://github.com/asyncapi/server-api/issues/50) in projects where we publish them

Reviewer: ```suggestion
- [Harsh Mishra](https://www.linkedin.com/in/harshcasper/) introduced a [CI workflow that validates docker images](https://github.com/asyncapi/server-api/issues/50) in projects where we publish them.
```
better for accessibility

> - [Abhijeet Jejurkar](https://github.com/abhijeetjejurkar) helped [improving Kubernetes deployment](https://github.com/asyncapi/server-api/issues/57) for our `server-api` service.

Reviewer: ```suggestion
- [Abhijeet Jejurkar](https://github.com/abhijeetjejurkar) [improved the Kubernetes deployment for our `server-api` service](https://github.com/asyncapi/server-api/issues/57).
```
accessibility :D

> - [Samriddhi](https://github.com/Samridhi-98) contributed [new `/bundle` endpoint](https://github.com/asyncapi/server-api/issues/55) to our `server-api` service.
> - [Ritik Rawal](https://github.com/ritik307) contributed [new `/diff` endpoint](https://github.com/asyncapi/server-api/issues/56) to our `server-api` service.
> - [Everly Precia Suresh](https://github.com/everly-gif) added a possibility to [host API reference docs](https://github.com/asyncapi/server-api/issues/43) for our `server-api` service through a dedicated endpoint.

Reviewer: ```suggestion
- [Samriddhi](https://github.com/Samridhi-98) contributed a [new `/bundle` endpoint to our `server-api` service](https://github.com/asyncapi/server-api/issues/55).
- [Ritik Rawal](https://github.com/ritik307) contributed a [new `/diff` endpoint to our `server-api` service](https://github.com/asyncapi/server-api/issues/56).
- [Everly Precia Suresh](https://github.com/everly-gif) added a possibility to [host API reference docs for our `server-api` service](https://github.com/asyncapi/server-api/issues/43) through a dedicated endpoint.
```
**guess** 😹

Author: I approve the change, as the intention is good, but I have doubts if `[new /bundle endpoint to our server-api service]` is better than `[new /bundle endpoint]`. We focus on accessibility for screen readers but forget how unreadable the content becomes for non-screen readers (almost entire text becomes a highlighted text, not good experience when reading content). I think sometimes we need to balance things

It is also super important to mention [Abir Pal](https://twitter.com/imabptweets), who helped to coordinate efforts and was the first point of contact for the community coming from OpenForce channels.

### Google Season of Docs

> Success with OpenForce was followed by unexpected announcement that we were accepted as an organization for Google Season of Docs (GSoD) with $10k budget :muscle:

Reviewer: ```suggestion
Success with OpenForce was followed by the unexpected announcement that AsyncAPI was accepted as an organization for Google Season of Docs (GSoD) 2022, with a $10k budget. :muscle:
```

<TwitterTweetEmbed
  tweetId='1514670329246801926'
  options={{
    cards: 'hidden',
    width: 200,
    maxWidth: 400
  }}
/>

> The amazing success of [Alejandra Quetzalli](https://twitter.com/QuetzalliAle) and an excellent opportunity for AsyncAPI Initiative to improve documentation big time!

Reviewer: ```suggestion
An amazing success for [Alejandra Quetzalli](https://twitter.com/QuetzalliAle) and an excellent opportunity for the AsyncAPI Initiative to improve its documentation big time!
```

> In mid-May, we will have the names of six interns we want to hire to work on two different projects to have better docs. Alejandra leads a super challenging task to interview as many candidates as possible (around 200!). Keep your fingers crossed.

Reviewer: ```suggestion
On May 16, we will announce the names of six interns we want to hire to work on two different projects to have better docs. Alejandra leads a super challenging task to interview as many candidates as possible (around 150!). Keep your fingers crossed.
```

> Alejandra and I will be mentors for interns for the next few months. Stay tuned to watch the progress.

Reviewer: ```suggestion
Alejandra and I will be mentors for our GSoD interns in the following months. Stay tuned to watch the progress.
```

### AsyncAPI Mentorship

> Even though the Google Summer of Code (GSoC) application was declined, we did not want to give up. We knew many community members joined and started contributing because they counted on AsyncAPI being part of GSoC. 

Reviewer: ```suggestion
We did not want to give up even though the Google Summer of Code (GSoC) 2022 application was declined. We knew many community members joined and started contributing because they counted on AsyncAPI being part of GSoC. 
```

> Well tl;dr our TSC just accepted an idea to start our own [AsyncAPI Mentorship](https://github.com/asyncapi/community/discussions/284) program :rocket:

Reviewer: ```suggestion
Well tl;dr, our TSC just accepted an idea to start our own [AsyncAPI Mentorship](https://github.com/asyncapi/community/discussions/284) program! :rocket:
```

> Now it is time until the 15th of May to discuss with mentors [in related ideas](https://github.com/issues?q=is%3Aopen+org%3Aasyncapi+label%3Agsoc) to understand the scope and clearly state you are a candidate for a given idea. Then we will ask TSC to vote and select 10 ideas that AsyncAPI Initiative should sponsor this year.

Reviewer: ```suggestion
Now we have until May 15 to discuss with mentors your [mentorship ideas](https://github.com/issues?q=is%3Aopen+org%3Aasyncapi+label%3Agsoc), understand the scope of each idea, and state clearly for which idea you are a candidate. Then we will ask TSC to vote and select 10 ideas that the AsyncAPI Initiative should sponsor this year.
```

## AsyncAPI Training

> ICYMI, we run an effort to create official AsyncAPI training videos. [Barbaño González](https://www.linkedin.com/in/barbano-gonzalez-moreno/) leads this topic and almost finished [scripts and storyboards](https://github.com/asyncapi/training/pulls) for the first set of videos.

Reviewer: ```suggestion
ICYMI, we run an effort to create official AsyncAPI training videos. [Barbaño González](https://www.linkedin.com/in/barbano-gonzalez-moreno/) leads this topic and almost finished [training scripts and storyboards](https://github.com/asyncapi/training/pulls) for the first set of videos.
```

> I do not think it is a secret if I say that great content is created only with great input, feedback, and review from people waiting for it. Please join Barbaño's efforts with any help as she is not doing it for herself but for the community.

Reviewer: ```suggestion
I do not think it is a secret to say that great content is created only with great input, feedback, and review from people waiting for it. Please join Barbaño's efforts with any help you can, as she is not doing it for herself but the community.
```

I recommend you [follow Barbaño's monthly updates](https://gist.github.com/Barbanio), especially if you want to participate.

## Docs Feedback Form

> One of our missions is to have kick-ass documentation. Easy right? 

Reviewer: ```suggestion
One of our missions is to have kick-ass documentation. Easy, right? 
```

> Writing docs is easy, just like writing code. Well, we all know the devil is hidden in details. Details like gathering requirements, figuring out architecture, and making sure you have a well-motivated group of people working on it. And last but not least, make sure the community can easily share feedback on your work transparently.

Reviewer: ```suggestion
Writing docs is easy, just like writing code. (Kidding!) Well, we all know the devil is hidden in the details. Details like gathering requirements, figuring out architecture, and making sure you have a well-motivated group of people working on it. And last but not least, make sure the community can easily share feedback on your work transparently.
```

Author: hmmm, I wasn't `Kidding` in a sense of making a joke. I do believe writing docs and code is easy. But what is complicated is all the work around, figuring out architecture, research, cooperation, etc. Writing final document or JS code is the easiest part in the whole process

Reviewer: Interesting.. writing code or documentation is not easy for everyone, so publishing this sentence could come across as minimizing how hard it is for others. I still think you need to leave this as a joke so that it does not come across as discouraging to beginners or people who still struggle ♥️

Author: but I can't change my mind, my opinion just because there might be someone that have a different opinion. This is not an article about writing docs or code, not for beginners in this subject. This is an update for the community on things that happened

Reviewer: That's fine, then make sure you start saying, **"For me, writing docs is easy, just like writing code."**

It's important to emphasize again this is your opinion because even tho this is a personal blog post, this section still reads like a statement.

> I wrote already about [Alejandra Quetzalli](https://twitter.com/QuetzalliAle) and the efforts to get AsyncAPI into GSoD. Getting a well-motivated group of people to work on docs, checked :white_check_mark:

Reviewer: ```suggestion
I wrote already about [Alejandra Quetzalli](https://twitter.com/QuetzalliAle) and the efforts to get AsyncAPI into GSoD. Getting a well-motivated group of people to work on docs, checked. :white_check_mark:
```

> New docs information architecture is [almost here too](https://github.com/asyncapi/website/pull/601). I think we can say "checked" :white_check_mark: to this one too.

Reviewer: ```suggestion
Our [new docs information architecture is almost here too](https://github.com/asyncapi/website/pull/601). I think we can say "checked" :white_check_mark: to this one too.
```

> Since this week, we can also say "checked" :white_check_mark: to the idea of getting feedback from the community.

Reviewer: ```suggestion
Since this week, we can also say "checked" :white_check_mark: to the idea of **getting docs feedback** from the community.
```

<TwitterTweetEmbed
  tweetId='1518596333749391363'
  options={{
    width: 200,
    maxWidth: 400
  }}
/>

> Each documentation page has a dedicated feedback form that one can fill in anonymously. As a result, feedback lands in [GitHub Discussion](https://github.com/asyncapi/community/discussions/340).

Reviewer: ```suggestion
Each documentation page has a dedicated feedback form that one can fill in anonymously. Your [docs feedback is injected anonymously into a Docs GitHub Discussion, as you can see here in this discussion example](https://github.com/asyncapi/community/discussions/340).
```
The anchor text in this case did not adequately explain what the link it's pointing to is about. That's why I made a point to re-phrase this one a little more.

Author: This is not a good change. It will look bad 👇🏼 

![Screenshot 2022-04-27 at 10 03 23](https://user-images.githubusercontent.com/6995927/165471125-7b375c46-b7c7-4382-baec-95614bd39003.png)

my understanding of accessibility focused anchors is to make sure they describe what resource is hidden under a link and not "summarizes" the implementation. Also `anonymously` is repeated again

Reviewer: I would go ahead and rephrase it and think about how you can still improve it :)

Your original implementation is still not good for accessibility so we still need to make it better ♥️

Author: > Each documentation page has a dedicated feedback form that one can fill in anonymously. Your feedback is passed into a docs-related GitHub Discussion, as you can [see here in this discussion example](https://github.com/asyncapi/community/discussions/340).

how about this?

Reviewer: Each documentation page has a dedicated feedback form that one can fill in anonymously. The form injects the feedback into a [docs-related GitHub Discussion like this example](https://github.com/asyncapi/community/discussions/340).
 
___ 

Close, how about this @derberg? It's still unclear for screenreader users who land/skip through that link ("see here in this discussion example") because it does not clarify its kind of discussion example. I think that clarifying the discussion example is needed for this anchor text to be more accessible. :)

Author: like it! changed

Huge applause to:
- [Alejandra Quetzalli](https://twitter.com/QuetzalliAle) for leading the change
- [Missy Turco](https://twitter.com/missyturco) for amazing support with design
- [Maciej Urbanczyk](https://github.com/magicmatatjahu) for detailed review

> - [Akshat Nema](https://twitter.com/AksNema) for implementation. Akshat demonstrated a lot of patience here. As an individual contributor, he waited for the merge for quite some time. There was a long discussion, many parties involved. Many contributors lost interest in such long-running PRs. Not Akshat :muscle:

Reviewer: ```suggestion
- [Akshat Nema](https://twitter.com/AksNema) for implementation. Akshat demonstrated a lot of patience here. As an individual contributor, he waited for the merge for quite some time. There was a long discussion, many parties involved. Many contributors lost interest in such long-running PRs. Not Akshat! :muscle:
```

One of our missions is to have kick-ass documentation. Easy, right? 

> Writing docs is easy, just like writing code (at least in my opinion :stuck_out_tongue_winking_eye:). Well, we all know the devil is hidden in details. Details like gathering requirements, figuring out architecture, and making sure you have a well-motivated group of people working on it. And last but not least, make sure the community can easily share feedback on your work transparently.

Reviewer: OMG THE BEST EMOJI ever, this is even awesomer now... YOURE WELCOME :D

Author: Thank you ❤️ 

https://www.youtube.com/watch?v=_ylepnCc9BE
